src/app/shared/helpers/null-undefined-predicates.ts

```diff
@@ -10,3 +10,10 @@ export function isNotNull<T>(e: T|null): e is T {
 export function isNotUndefined<T>(e: T|undefined): e is T {
   return e !== undefined;
 }
+
+export const isDefined = isNotUndefined;
+
+export function ensureDefined<T>(value: T | undefined, message = 'expected value to be defined'): T {
```

Comment:
the error message should probably be "expected value to not to be undefined"

Reply:
Fixed